packages/lsd-react/src/components/TabItem/TabItem.styles.ts

```diff
@@ -35,6 +41,10 @@ export const TabItemStyles = css`
     }
   }
 
+  .${tabItemClasses.tabWithoutIcon} {
+    justify-content: center;
+  }
+
```

Comment:
Ihor said the tabs' content should be centered when there's no icon.

Reply:
Sorry for the late response. I've implement your suggestion @jeangovil - anything else let me know :pray: